### DISCUSSION TITLE
Search - Hide yanked and outdated crates by default

### DISCUSSION THREAD

**Discussion by Mazwak** [rust-lang/crates.io#4703] [Feature Requests]
When searching crates.io by relevance, some results are not (relevant) at all.

For example, if you search 'xlsx', the first page has :
* 2 yanked crates
* 5 outdated crates (> 2 years with no updates), 3 without even a README
* 3 'OK' crates

I don’t see any reason to return yanked crates (by default).

And a max age would be nice, maybe higher than 2 years though.

**Comment by carols10cents** [comment]
See https://github.com/rust-lang/crates.io/issues/145, this is definitely something we want that someone needs to design and implement!
